agent/src/main/java/com/teamscale/jacoco/agent/commit_resolution/git_properties/GitPropertiesLocator.java

```diff
@@ -30,21 +31,26 @@
 	private final DelayedUploader<T> uploader;
 	private final DataExtractor<T> dataExtractor;
 
-	public GitPropertiesLocator(DelayedUploader<T> uploader, DataExtractor<T> dataExtractor) {
```

Comment:
Done (FYI, I keep discussion thread open because when I do reviews, I like to look at the resolution of the previous comments and that difficult if I don't see them anymore when they are resolved)